package-lock.json

```diff
@@ -1,13558 +1,8 @@
 {
   "name": "@ibm-cloud/platform-services",
   "version": "0.22.10",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
It looks like you are using npm v6 because the package-lock.json file was changed back to being a "lockfileVersion" 1 file.
Please revert these changes to package-lock.json, then upgrade your npm to be 8.x (I'm using 8.5.4 on a Fedora Linux laptop).  Then if you run `npm install` locally and need to commit changes made to package-lock.json, it will at least be the same lock-file version.
Alternatively, you could avoid making changes to package-lock.json altogether if you first revert your package-lock.json changes so we go back to the previous version, and then just run `npm ci` locally, and that will load exactly the set of dependencies (and the specific versions) contained in package-lock.json.